app/api/session_chat_messages.py

```diff
@@ -0,0 +1,103 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from flask import request, current_app as app
+from flask_jwt import current_identity as current_user, _jwt_required
+
+from app.models.session import Session
+from app.api.helpers.db import safe_query
+from app.api.helpers.permission_manager import has_access
+from app.api.helpers.exceptions import ForbiddenException
+from app.api.helpers.utilities import require_relationship
+from app.api.schema.session_chat_messages import SessionChatMessageSchema
+from app.models import db
+from app.models.session_chat_message import SessionChatMessage
+from app.models.speaker import Speaker
+from app.models.ticket_holder import TicketHolder
+
+
+class SessionChatMessageListPost(ResourceList):
+    """
+    Create Session Chat Messages
+    """
+    def before_post(self, args, kwargs, data):
+        """
+        before post method to check for required relationship and proper permission
+        :param args:
+        :param kwargs:
+        :param data:
+        :return:
+        """
+
+        if 'Authorization' in request.headers:
+            _jwt_required(app.config['JWT_DEFAULT_REALM'])
+        else:
+            raise ForbiddenException({'source': ''}, 'Authorization required')
+        require_relationship(['session'], data)
+
+        data['user_id'] = current_user.id
+        session = db.session.query(Session).filter_by(id=int(data['session'])).first()
+        event_id = session.event_id
+        attendee = db.session.query(TicketHolder).filter_by(event_id=event_id, email=current_user.email)
```

Comment:
Yes it works.
adding first() allows us to later get the field values of that record like `attendee.website_url`
but since we just want that is query returns a record so adding .first() or not doesn't change any thing.